plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java

```diff
@@ -241,6 +242,7 @@ public void getPluginDownloadUrlTest() {
         VersionNumber otherVersion = new VersionNumber("otherversion");
 
         plugin.setVersion(otherVersion);
+        plugin.setGroupId("");
```

Comment:
is this needed?

Reply:
Yes, the way I currently have it set up. I'm reusing the same plugin to test, but changing it's attributes. Because I'm checking the groupId to determine if a requested plugin should be downloaded from the incrementals repository, this needs to be set back to null or an empty string if the plugin should not be downloaded from an incrementals repository.

Reply:
https://github.com/jenkinsci/plugin-installation-manager-tool/blob/d17bc5269a901e22cd2858c426525472bea535b8/plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java#L441

Reply:
probably better to create a new plugin instance, it would be clearer, and you can set all the args in the constructor?

perhaps also better to create a separate test case

Reply:
Fixed in 48098fd.